tests/e2e/server/data.js

```diff
@@ -0,0 +1,44 @@
+export const initial = {
+  authId:
+    'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhdXRoSW5kZXhWYWx1ZSI6IlVzZXJuYW1lUGFzc3dvcmQiLCJvdGsiOiJ1ZWRkbW83NW9iZ2YwZjY4amw5Z3JoNnBrdSIsImF1dGhJbmRleFR5cGUiOiJzZXJ2aWNlIiwicmVhbG0iOiIvIiwic2Vzc2lvbklkIjoiKkFBSlRTUUFDTURJQUJIUjVjR1VBQ0VwWFZGOUJWVlJJQUFKVE1RQUNNREUuKmV5SjBlWEFpT2lKS1YxUWlMQ0pqZEhraU9pSktWMVFpTENKaGJHY2lPaUpJVXpJMU5pSjkuWlhsS01HVllRV2xQYVVwTFZqRlJhVXhEU214aWJVMXBUMmxLUWsxVVNUUlJNRXBFVEZWb1ZFMXFWVEpKYVhkcFdWZDRia2xxYjJsYVIyeDVTVzR3TGk1MVpGSnNaV1ZJVEVkS2EybzBNVkZPVnpKQ2VVNUJMbVkyTVZGb2QySkhNMWhwU2t0MWEzaDVUamR3TmxSdlVIWklZVTluYkZad2JYWlJTbU16T1hrNFUwRk5lbGd0VXpSSExWUnVYM1pMTFhwVFZFeEhZVUZQU3pCek9Xa3diSE5EUjJKdU5HZFFia1JSYkVaTlNqZE1jRW95WDJwZmRFTTBTVGszWlY5b2VGQmtXRXc1WTJGQ1F6UklhWEJVWTFveWFWTTFPVXR6UlZSYWVXUnRTMWx2WlVObWVreE5iREZHWTNsV04zbDZNMWs1ZVVSb05sOURTMWx3VWxsaFVrcEtiUzB4VTNOZlkweENWblJaVGs0d1RYUjJaMTgyVUVaVGRrdFplWEp5TFU1blpXRTBTemgwVFdGdFJreElPVkZUVEhKbVMyUTVPV0ZaVW5KYVUycGtSV1pEYkd4blNtMW5PV2hxTW1OM1Jqa3dhMTk2YlhCbVN6TjJTVVZvUmt3elIxTnZZbWgyYVdOcVNETkNjbEp4V0hCVFprTk9lRGRIUjFrd1NtNDJUWHBwTVVZNFpWVnpUMDU2WjA5b1ZqWnlkblpvUkdSblVtNDBMVEpyYmxsYVVGUkplamRDYmtkc1lXeHBVREIxZWtoQ01XSmtiRU5vY1V0bGNrOVRRalJHTFRkR2J6Rm9kVE4yUm1wa2FsTlNSa00wZEcxc2RrYzRURk5HZFhOSGJWTm1ZMjlmTjNCNlJGQmZZalF4ZEZoblJGSk5ORkppUmsxVVNteEtWVU5uWDI1SFNIWjZVRmN3ZFVaUVVFTm5XVXRPUzI5d1RrUTJPSE5FT1Vob05XcG1VVlp4YVdORU1DMHhjR1pIUjAxSVIzQnNYM2xZVlhsVVMzUnNjalZRY1MxT2FrVTJTbGR0ZW1ReWIweGtRMFEwUVdWd1FrVmxhM1ZXTVZWRmQxZEVNVTA0ZDJNemQzQkJWbVI2YkdOdk5tYzNkblZ1UkVFNVdXb3dibkJGVW5sVFpXVjRhVWRmTVVOTVZIQTJaMkV3ZDJkSFZUUjZObTVWV2tSZlNqVjVRbHBqUm1aMmFXaHRjVUl5Vm1Ka056SlVjbUo1TkZwV1REWlhOMU0zUVd0dU1ETjFRVWxIUld3eVNESkNRV0UyWkZaeWNIUnNWbUk1YldWcU5WQjBVV1J5VTBrNGFFVjRlalI2TVhoTlMySndSMHBNVkRJMmVVWlZSbFZyWXprd1JuQkpMa0p3WjJ0M01HdFZlbnBGVUVST2NsOHRVbGxKTlVFLkpfY2FydHB3X1JCZloxbFNrSEJvVTZmemVXUzRZM0RSLWVPQ0xidFRzNmsiLCJleHAiOjE1NzU2NzkyMjIsImlhdCI6MTU3NTY3ODkyMn0.f9depsMggE-iV_u2BDZNV8aIm6YKoIgzyTBDifKZRJc',
```

Comment:
Feel free to reduce token strings like `authId`,  `tokenId`, etc for readability since they're opaque.  Could even be `authId: 'foo'`.

Reply:
Help, good idea. I'll clean this stuff up.